### PR TITLE
Jolo branch

### PR DESCRIPTION
**Pull request by jolo-dev**
Add Stackedbar.js and put example in the index-Files.

### CODE REVIEW

**Comment by jwilber** [comment]
Super stoked to see this, will review tonight!

**Comment by jwilber** [comment]
Hey @jolo-dev - this looks amazing! Well done!

There is an issue with the hover effects. Namely, paths are only highlighted when the mouse is directly above a path, but we want those paths to highlight when you're over _any_ portion of the rectangle's inner area.

The fix for this issue is creating a stacked bar graph in the `addInteraction()` code using regular D3 that sits in the exact same place as the `roughjs` plot.  That way, when you call mouseover events from `selectAll(this.interactionG)...`, everything hovers at once.

You can see that this is the case for the other plots (e.g. `Bar.js` by changing the `.attr('fill', 'transparent');` line in the `addInteraction()` code to something like `.attr('fill', 'green');`:

Screen Shot 2019-11-06 at 4 39 59 PM

As you can see, under the hood we have a bar chart in the same position as the `rough.js` version.

So basically, we need to create that stackedbar chart in the background. I imagine it will look pretty similar to what you have in defined in the `stacked()` function, but if you want me to help work on this let me know.

Very stoked on how it looks!


tl;dr, the code in `addInteraction()` should create a stacked bargraph in the background of the `rough.js` version. Currently, it's not doing that, so the current hover effects are very choppy.

**Comment by jolo-dev** [comment]
Hey @jwilber,
Thanks for your reply. Much appreciated, I was on vacation lately and I am back on track ;)
However, I need to catch up on work and will come back to this soon.

Thanks again!

**Comment by jolo-dev** [comment]
Hi @jwilber,

I've apply the changes. Could you have a look?

Best,

**Comment by jwilber** [comment]
Very awesome work @jolo-dev, thanks a lot!

**Comment by jwilber** [comment]
PS - I’ve listed you as a contributor in the readme, let me know if you’d like to have me link to a website instead of your github page

**Comment by jolo-dev** [comment]
Awesome! I have no problem with that. I am also building a VueJS-Wrapper.
So stay tuned!
